compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirInapplicableLateinitChecker.kt

```diff
@@ -33,7 +33,7 @@ object FirInapplicableLateinitChecker : FirMemberDeclarationChecker() {
         primitives = this
     }
 
-    override fun check(declaration: FirMemberDeclaration, context: CheckerContext, reporter: DiagnosticReporter) {
+    override fun check(declaration: FirVariable<*>, context: CheckerContext, reporter: DiagnosticReporter) {
```

Comment:
Here we also can use `FirPropertyChecker`

Reply:
Yep. I also found many more cases where using aliased checker can remove unnecessary declaration filtering.